.gitsecret/paths/mapping.cfg

```diff
@@ -0,0 +1,2 @@
+.env:1287489e9d838c379d3bdff223da342eab1ebdcc182dbac2e4c9cf723b84cf9d
```

Comment:
Hi @123MwanjeMike are these supposed to be committed?

Reply:
Yes @NoahNsimbe.
These are hashes only people with whom the secret have been shared are able to use to decrypt the file.
No worries.